transforms/csdl/preprocess_csdl.xsl

```diff
@@ -786,6 +786,25 @@
     <xsl:template match="edm:Schema[@Namespace='microsoft.graph']">
         <xsl:copy>
             <xsl:apply-templates select="@* | node()"/>
+            
+            <!-- Remove navigability for driveItem/workbook navigation property for DevX API-specific CSDL-->
+            <xsl:choose>
+                <xsl:when test="$remove-capability-annotations='False'">
```

Comment:
assuming this is true for kiota based SDKs?

Reply:
OpenApi generation for Kiota based SDKs does include annotations so this navigability will also be removed for Kiota based SDKs. 
To omit the file for Kiota based SDKs we may need to include the `open-api-generation` variable in the test [here ](https://github.com/microsoftgraph/msgraph-metadata/blob/master/transforms/csdl/preprocess_csdl.xsl#L11)

Reply:
Thanks for the context. I believe kiota deceptions should not have the restriction. Irvine, can you make the change please?

Reply:
This is effected.